src/server_manager/web_app/gallery_app/main.ts

```diff
@@ -114,12 +126,12 @@ export class TestApp extends LitElement {
     this.setLanguage('en');
   }
 
-  async setLanguage(newLanguage: string) {
+  setLanguage(newLanguage: string) {
```

Comment:
Why are you changing this?

Reply:
`outline-server-view` depends on having `this.language` correct at startup.  The asynchronous `setLanguage` call meant that `this.language` was `''` when `outline-server-view` loads, resulting in errors and failure to load.

Reply:
Could you add a comment. It's not clear.

Also, this seems like a bug in the outline-server-view. All components should be able to have their language changed.

An alternative is to place the `<outline-server-view>` inside an if-block that tests if the language and localize are present.

Reply:
The main issue is that `outline-server-view` makes heavy use of `this.localize` and `this.language`, resulting in uncaught exceptions because these are not initialized yet.  (Polymer somehow tolerates this when they are used in a template.)

I've reverted this change, and instead made the relevant Polymer elements tolerant of null/empty `localize` and `language` properties.  I've also added various missing bindings so that changes are tracked correctly.

Reply:
I don't get why this wasn't a problem before. Isn't there a default localize function set? If not, can't we set a dummy localize in the app root constructor?

I see you are passing the localize function in some template calls that depend on it, which is a good fix.

For the unit stuff, check the presence of the language in the caller instead (like we already do in some places), so we don't hide any bugs. It's up to the caller to decide what to do in those cases.